### PR TITLE
feat: zendesk contact form setup

### PR DESCRIPTION
**Pull request by ItsANameToo**
## Summary

This adds zendesk to the contact form to allow tickets to be created.

It's based on the contact form from foundation to keep it in line with the rest of the projects. We should look into changing the contact form to a livewire component later on.

Make sure to set the following `.env` variables:
```
ARKSCAN_SUPPORT_ENABLED=true
ZENDESK_SUBDOMAIN=
ZENDESK_USERNAME=
ZENDESK_TOKEN=
```

TODO:
- [x] hide support when `.env` var is set to disabled
- [x] add `support` entry in navbar
- [x] tests
- [x] phpstan

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: app/Providers/AppServiceProvider.php [hunk 1]
```diff
@@ -42,6 +43,9 @@ public function boot()
         $this->registerDataBags();
 
         Fortify::loginView(fn () => abort(404));
+
+        $navigationEntries = $this->navigationEntries();
+        View::composer('layouts.app', static fn ($view) => $view->with(['navigationEntries' => $navigationEntries]));
```
Comment (alexbarnsley) [inline app/Providers/AppServiceProvider.php:48]: Can't you do this? :thinking: 

```suggestion
        View::composer('layouts.app', static fn ($view) => $view->with(['navigationEntries' => $this->navigationEntries()]));
```

Comment (ItsANameToo) [reply app/Providers/AppServiceProvider.php:48]: no `$this` in that static context

Comment (alexbarnsley) [reply app/Providers/AppServiceProvider.php:48]: I see :thinking: fair enough :oleglala: I figured `$this` would pull from the above above in an inline method

**Review by alexbarnsley — APPROVED** [review]

File: app/Providers/AppServiceProvider.php (hunk 1 above)
Comment (ItsANameToo) [reply app/Providers/AppServiceProvider.php]: can remove static :trollface: thanks

**Comment by crnkovic** [comment]
https://app.clickup.com/t/31gr99y

**Pull request merged**